### PR TITLE
Migrations: migrate to admin container v0.9.2

### PR DESCRIPTION
**Pull request by jpculp**
**Description of changes:**

Update and migrate from admin container v0.9.0 to v0.9.2.

**Container changes:**

Remove /etc/config/selinux from image.

**Testing done:**

Tested in #2472, thanks @ecpullen!

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Comment by jpculp** [comment]
Added a comma to hopefully avoid folks needing to rebase after.

**Review by ecpullen — APPROVED** [review]
LGTM

File: sources/models/shared-defaults/aws-host-containers.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ superpowered = true
 
 [metadata.settings.host-containers.admin.source]
 setting-generator = "schnauzer settings.host-containers.admin.source"
-template = "{{ ecr-prefix settings.aws.region }}/bottlerocket-admin:v0.9.0"
+template = "{{ ecr-prefix settings.aws.region }}/bottlerocket-admin:v0.9.2"
```
Comment (arnaldo2792) [inline sources/models/shared-defaults/aws-host-containers.toml:7]: Nit: Probably a note in the commit (and description) about why we updated from `0.9.0` to `0.9.2` could help :D!

**Review by arnaldo2792 — APPROVED** [review]

**Review by gthao313 — APPROVED** [review]

File: sources/models/shared-defaults/aws-host-containers.toml (hunk 1 above)
Comment (jpculp) [reply sources/models/shared-defaults/aws-host-containers.toml:7]: I think that would get rather verbose. In general we should just treat this as a typical dependency bump and I suspect we're going to update our host containers more regularly going forward.

Comment (jpculp) [reply sources/models/shared-defaults/aws-host-containers.toml:7]: Added **Container changes** to the description though. 🚀

**Pull request merged**